chunks 1,4-6: Set base paragraph indentation to 5n (upstream commit 5d2e49f8).
chunk 2: Tweak section headers (local change).
chunk 3: Set the default operating system name (local change).

Index: tmac/an.tmac
--- tmac/an.tmac.orig
+++ tmac/an.tmac
@@ -170,7 +170,7 @@
 .
 .de an-reset-margin-and-inset-level
 .  nr an-inset-level 1
-.  nr an-margin \\n[IN]
+.  nr an-margin \\n[BP]
 .  nr an-saved-margin1 \\n[IN]
 .  nr an-prevailing-indent \\n[IN]
 .  nr an-saved-prevailing-indent1 \\n[IN]
@@ -195,7 +195,7 @@
 .  ds an*section1 General Commands Manual\"
 .  ds an*section2 System Calls Manual\"
 .  ds an*section3 Library Functions Manual\"
-.  ds an*section4 Kernel Interfaces Manual\"
+.  ds an*section4 Device Drivers Manual\"
 .  ds an*section5 File Formats Manual\"
 .  ds an*section6 Games Manual\"
 .  ds an*section7 Miscellaneous Information Manual\"
@@ -306,7 +306,7 @@
 .
 .  ds an-extra1 "\\$3\"
 .  ie (\\n[.$] > 3)    .ds an-extra2 "\\$4\"
-.  el                  .ds an-extra2 \" empty; but .AT/.UC can override
+.  el                  .ds an-extra2 "OpenBSD\"
 .
 .  if '\\*[an-extra1]'' \{\
 .    ds an-msg .\\$0 missing third argument; suggest document\"
@@ -681,7 +681,7 @@ contains unsupported escape sequence
 .  an-break-paragraph
 .  an-reset-margin-and-inset-level
 .  fi
-.  in \\n[an-margin]u
+.  in \\n[BP]u
 .  ti 0
 .  nr an-devtag-needs-end-of-heading 1
 .  DEVTAG-SH 1
@@ -708,7 +708,7 @@ contains unsupported escape sequence
 .  an-break-paragraph
 .  an-reset-margin-and-inset-level
 .  fi
-.  in \\n[IN]u
+.  in \\n[BP]u
 .  ti \\n[SN]u
 .  nr an-devtag-needs-end-of-heading 1
 .  DEVTAG-SH 2
@@ -1323,6 +1323,10 @@ contains unsupported escape sequence
 .  if !r PDFOUTLINE.FOLDLEVEL .nr PDFOUTLINE.FOLDLEVEL 1
 .  if !r PDFHREF.VIEW.LEADING .nr PDFHREF.VIEW.LEADING 10p
 .\}
+.
+.\" base paragraph indentation
+.if !r BP \
+.  nr BP 5n
 .
 .\" continuous rendering (one long page)
 .if !r cR \{\
